engine/src/main/scala/cromwell/engine/workflow/tokens/TokenQueue.scala

```diff
@@ -17,7 +18,7 @@ import scala.collection.immutable.Queue
 final case class TokenQueue(queues: Map[String, Queue[TokenQueuePlaceholder]],
                             queueOrder: Vector[String],
                             eventLogger: TokenEventLogger,
-                            private [tokens] val pool: UnhoggableTokenPool) {
+                            private [tokens] val pool: UnhoggableTokenPool) extends LazyLogging {
```

Comment:
Out-of-curiosity: After coming across a couple links that lean towards it, I've started using `StrictLogging` in most places. Any particular reason this class uses `LazyLogging`?

One such discussion: https://stackoverflow.com/questions/41648984/what-is-the-difference-between-scala-strictlogging-and-lazylogging/42846001#42846001

Reply:
No good reason so `StrictLogging` sounds fine to me